.github/actions/do_build_ock/action.yml

```diff
@@ -98,69 +95,80 @@ inputs:
     description: "Disable unitcl vecz checks"
     default: OFF
   gtest_launcher:
-    description: "Googletest suite launcher command"
+    description: "Googletest suite launcher command (default ubuntu)"
```

Comment:
what does default ubuntu mean?

Reply:
what you have doesn't make that clear, perheps say 'default launcher used for ubuntu'